dali/driver/base.py

```diff
@@ -234,3 +236,66 @@ def close(self):
         self._disconnecting = True
         self._stop_listening.set()
         super(USBListener, self).close()
+
+class SerialBackend(Backend):
+    """``Backend`` implementation for Quad-DALI-USB-Interface. Uses serial connection."""
```

Comment:
Please update the docs, backend is not Quad-DALI bound.

Reply:
👍